.github/workflows/notebook-local-verify.yaml

```diff
@@ -0,0 +1,38 @@
+name: Docker notebook tests
```

Comment:
nit:

```suggestion
name: Test notebooks with LocalProvider
```

or soemthing like that

Reply:
was already done before I hit "comment" :smile:

Reply:
I've just noticed.